app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -62,17 +55,28 @@ class Form526DocumentUploadFailureEmail < Job
           }
         )
         raise e
+      ensure
+        StatsD.increment("#{STATSD_METRIC_PREFIX}.exhausted")
```

Comment:
Good call, this is already right above the rescue block but it might not always be and we do want to ensure it increments